setup.py

```diff
@@ -20,8 +20,9 @@
     author_email="stac@radiant.earth",
     url="https://github.com/stac-utils/pystac",
     packages=find_packages(),
+    package_data={p: ["py.typed"] for p in find_packages()},
```

Comment:
```suggestion
    package_data={"": ["py.typed"]},
```

This will ensure that `py.typed` is included for all packages included in the distribution.

Reply:
Ahh, when I was working out how to do it in a `setup.cfg` for `stactools` I saw something about `"" keys matching all` and had no idea what it was talking about :laughing:. Thanks!